debugger/command_line.ml

```diff
@@ -1221,7 +1229,13 @@ It can be either :\n\
      { var_name = "break_on_load";
        var_action = boolean_variable false break_on_load;
        var_help =
-"whether to stop after loading new code (e.g. with Dynlink)." }];
+"whether to stop after loading new code (e.g. with Dynlink)." };
+      { var_name = "mapping";
+        var_action = environ_variable false "BUILD_PATH_PREFIX_MAP";
+        var_help =
+"BUILD_PATH_PREFIX_MAP used to map abstract\n\
+paths to runtime environment."}
```

Comment:
You could include a trailing semicolon here, so that the next person does not have to change line 1237 to add a new element.

Reply:
Done.